test/SourceKit/InterfaceGen/gen_clang_module.swift.response

```diff
@@ -112,18 +96,10 @@ public var fooIntVar: Int32
 
 /// Aaa.  fooFunc1.  Bbb.
 public func fooFunc1(_ a: Int32) -> Int32
-
 public func fooFunc1AnonymousParam(_: Int32) -> Int32
 public func fooFunc3(_ a: Int32, _ b: Float, _ c: Double, _ d: UnsafeMutablePointer<Int32>!) -> Int32
-
-/*
-  Very good
-  fooFuncWithBlock function.
-*/
 public func fooFuncWithBlock(_ blk: ((Float) -> Int32)!)
-
 public func fooFuncWithFunctionPointer(_ fptr: (@convention(c) (Float) -> Int32)!)
-
 public func fooFuncNoreturn1() -> Never
 public func fooFuncNoreturn2() -> Never
```

Comment:
No blank line between top-level decls doesn't seem to be expected (i.e. swift modules show blank lines). Looking into it

Reply:
There are some difference between Swift decl printing and clang decl printing in `swift::ide::printModuleInterface()`, Swift decls always have trailing newline, but clang decls don't. Fixing it is out of scope of this PR.